### ISSUE TITLE
Add 'meteor create --react'.

### ISSUE THREAD

**Issue by alexsicart** [meteor/meteor-feature-requests#182]
### Feature requests should include as much detail as possible:
- [ Add 'meteor create --react']
- [ As we know, create meteor app's is amazing, but we join React Facebook and Meteor, is the perfect combination]
- [ Create a flag with react option, when we want to create a new meteor app]

**Comment by abernix** [comment]
Thanks for opening this, @alexsicart! This spawned from discussion in [this tweet](https://twitter.com/abernix/status/910110665808994305) which mentions that there was scattered conversation about getting an official skeleton in https://github.com/meteor/meteor/issues/6974 and also starting at https://github.com/meteor/meteor/pull/8728#issuecomment-305069416 (warn: very large 1.6 release thread), but no official issue to track the importance.

**Comment by abernix** [comment]
I believe keeping the base `--react` application relatively simple and, feature-wise, on parity with the Blaze version would be my initial thought.  We should explore if we want [`--full`](https://github.com/meteor/meteor/tree/devel/tools/static-assets/skel-full) and [`--bare`](https://github.com/meteor/meteor/tree/devel/tools/static-assets/skel-bare) variants and what those should include.

And some server-side rendering (SSR) pattern should likely be available in one of these variants!

**Comment by alexsicart** [comment]
Yes I agreee with you, simple, without authentification. Only main.js, main.html, connected to the app.js in the components folder. Displaying Hello React.

**Comment by alexsicart** [comment]
This afternoon I'll modify https://github.com/alexsicart/Create-React-Meteor-App, to make it more simple.

**Comment by alexsicart** [comment]
https://github.com/alexsicart/Create-React-Meteor-App Skeleton idea

**Comment by iwojap** [comment]
+1 for all of this, including the --bare and --full versions.

**Comment by filipenevola** [comment]
Hello, do you think that we should include Apollo in this app example? Or maybe have one additional option with Apollo? 

I'm working in this repo https://github.com/CodeFTW/meteor-react-latest to keep an up to date working example of Meteor + React + Apollo. Also I'm trying to put good practices in place, like using imports folder and one example of each common operation (router, mutation, query, etc). 

What do you think? What is the best path to go and how can I help?

**Comment by hanimay** [comment]
Hello,
I assumed having react command would lead to something similar or a more integrated version to this repo : https://github.com/Urigo/meteor-client-bundler.

This way we can easily benefit from HTTP/2 and CDN for our clients as well as benefiting from Webpack.

Kindly advise.

**Comment by alexsicart** [comment]
@filipenevola It could be a flag! Meteor + React + Apollo. Good idea!!

**Comment by theodorDiaconu** [comment]
I think this will be a never-ending task. After a new structure is introduced, we'll need another boilerplate, then another, when a `git clone REPO` can be as easy as `meteor create --react`, maybe you will need a router, or something else. And after SSR matures you will want a --react-ssr, --react-router-ssr. 

Maybe create a convention to add as topic "meteor-boilerplate" to GitHub ?
https://github.com/topics/meteor-boilerplate

**Comment by dmihal** [comment]
Any update on this?

I'm giving a demo on Meteor + React in a month and I'd love to be able to start it with something like `meteor create --react-full`.

I'd be happy to put together a PR if people are supportive!

**Comment by StorytellerCZ** [comment]
@dmihal PR would be a great start.
